Cargo.toml

```diff
@@ -28,11 +28,13 @@ futures = "0.1.11"
 futures-cpupool = "0.1"
 hyper = { git = "https://github.com/hyperium/hyper", optional = true }
 hyper-tls = { git = "https://github.com/hyperium/hyper-tls", optional = true }
+jsonwebtoken = { version = "2.0", optional = true }
 libc = "0.2.10"
 local-encoding = "0.2.0"
 log = "0.3.6"
 lru-disk-cache = { path = "lru-disk-cache" }
 number_prefix = "0.2.5"
+openssl = { version = "0.9", optional = true }
```

Comment:
I'm adding a commit that removes openssl and uses pem-parser instead, although I noticed when doing so that it doesn't return an error if the PEM is invalid base64-- it `unwrap()`s it. Probably not a big deal-- I think it's unlikely that people will supply files that are the appropriate format but don't have a valid PEM.